libsel4/sel4_plat_include/tqma8xqp1gb/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021, Breakaway Consulting Pty. Ltd.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#ifdef HAVE_AUTOCONF
+#include <autoconf.h>
+#endif
+
+#define seL4_NumHWBreakpoints (6)
```

Comment:
It would be nice if you add a comment here where these values come from. I Assume this is basically the Cortex-A35 manual? So this is not really `tqma8xqp1gb` specific on the end. One day we might be able to put all this into core-specific files then to avoid code duplication

Reply:
Added reference.

Agree that it would be nice to have Cortex-A35 specific header files to take care of this, but I think that's beyond scope of this change.